cli-plugins/insight/triaging-vulnerabilities.hbs.md

```diff
@@ -0,0 +1,108 @@
+# Triaging vulnerabilities
+
+This topic describes how to add analyses to any vulnerability associated to a workload
+in the Supply Chain Security Tools - Store. Keep in mind this is an experimental feature, and the
+API is prone to changes in subsequent releases.
+
+## <a id='triage-description'></a>What is triaging?
+
+Vulnerability analysis, triaging or triage is the process of evaluating a reported vulnerability in
+order to determine an effective remediation plan. It is through triage that app teams are able to
+generate useful insights about the vulnerabilities in their software so that they can make the right
+decisions about when and how to mitigate them. The current implementation of triage follows
+[CycloneDX's Vulnerability Exploitability eXchange](https://cyclonedx.org/capabilities/vex/)
```

Comment:
Exchange looks like the capitalization is on the wrong letter

Reply:
That is intended, since that's the definition of VEX. I'll add the acronym to make it clearer